src/Microsoft.DocAsCode.HtmlToPdf/packages.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="iTextSharp" version="5.5.8" targetFramework="net4" />
+  <package id="PDFsharp" version="1.50.5147" targetFramework="net20" />
```

Comment:
Could you please explain the reason why change to use PDFsharp instead of iTextSharp?

Reply:
See https://github.com/dotnet/docfx/issues/4250. iTextSharp has a GNU AGPL 3.0 license and PDFSharp has a MIT license.

Reply:
That's great, thanks for your help.

Reply:
@partychen , do we need more testing before actually making the switch?